docs/labels.md

```diff
@@ -47,6 +47,8 @@ Via a _label selector_, the client/user can identify a set of objects. The label
 The API currently supports two types of selectors: _equality-based_ and _set-based_.
 A label selector can be made of multiple _requirements_ which are comma-separated. In the case of multiple requirements, all must be satisfied so comma separator acts as an AND logical operator.
 
+A collection of objects queried with an empty label selector, which is a selector containing zero key-value pairs, returns all objects in the collection.
```

Comment:
Perhaps this flows a bit better from the previous paragraph?

"An empty label selector (that is, one with zero requirements) selects every object in the collection."